util/io/src/main/scala/sbt/IO.scala

```diff
@@ -68,18 +86,29 @@ object IO {
   }
```

Comment:
Do you want to restore the semantics that were changed here as well?

https://github.com/sbt/sbt/commit/9fbbaab58e246bce7b21117e079206519c386b57?diff=unified#diff-dd0ad8518ad4dee1e8ab8f4f1d69cbe4R61